pkg/core/blockchain.go

```diff
@@ -1776,7 +1776,7 @@ func (bc *Blockchain) addHeaders(verify bool, headers ...*block.Header) error {
 	if len(headers) > 0 {
 		var i int
 		curHeight := bc.HeaderHeight()
-		for i = range headers {
+		for ; i < len(headers); i++ {
```

Comment:
It should be a regression of some commit and a comment would be handy for other `range` lovers.

Reply:
It's not a regression, it's actually the implementation itself introduced by https://github.com/nspcc-dev/neo-go/commit/c3e73c5b7d8abfd9418049112b4876a45b6ed990.